src/components/Error.jsx

```diff
@@ -0,0 +1,3 @@
+export const Error = () => {
+    return <h2>  ❌ Error has occured: Page not found ❌ </h2>
```

Comment:
I like the introduction of this page to handle invalid paths, I do get it when I click login as a user which isn't ideal.